app/qml/components/ProjectDelegateItem.qml

```diff
@@ -164,7 +164,7 @@ Rectangle {
                else root.secondaryColor
         horizontalAlignment: Text.AlignLeft
         verticalAlignment: Text.AlignBottom
-        elide: Text.ElideRight
+        elide: Text.ElideLeft
```

Comment:
That's one small step for man, one giant leap for "organizations-with-long-names"